python/cog/schema.py

```diff
@@ -66,6 +67,8 @@ class PredictionResponse(PredictionBaseModel):
     error: t.Optional[str]
     status: t.Optional[Status]
 
+    webhook: t.Optional[pydantic.AnyHttpUrl]
```

Comment:
I'm wondering if this actually needs to be in the response object? Or did it get added because we're trying to set a field that doesn't exist and that was bombing out?